packages/shared/src/contexts/FeedContext.tsx

```diff
@@ -173,8 +185,20 @@ export function FeedLayoutProvider({
     defaultFeedSettings,
   );
 
+  const finalSettings =
```

Comment:
Why do we need custom overwrites here?

Reply:
I think since we are only doing it for Laptop and we want a specific amount of cards to be displayed only.

However, I think we should not do this here. The update must happen from where the `currentSettings` is created. This is so that, anywhere else, the values returned by these settings, would be consistent across the board.

For example, we would be sending the same analytics events as if this was the actual Feed page. For that, we utilized values such as `virtualizedNumCards`, etc. which reference the values from these overridden settings.

Reply:
Nvm. This is already the root of the settings. we should be good.